test.js

```diff
@@ -1,24 +1,37 @@
-const path = require("path");
-process.env.FUSEBOX_DIST_ROOT = __dirname;
-process.env.FUSEBOX_VERSION = "2.4.0";
-process.env.PROJECT_NODE_MODULES = path.join(__dirname, "node_modules");
-process.env.SPARKY_LOG = false;
-process.env.LOGGING = false;
-process.env.DYNAMIC_IMPORTS_DISABLED = true;
-const { FuseBox } = require("./bin/fusebox");
+const { join } = require('path');
+const { argv, env } = require('process');
+const { FuseBox } = require('./bin/fusebox');
+
+const defaultTestFileMask = '*.test.ts';
+const projectRoot = __dirname;
+
+/*
+ * Definition of environment variables required to run tests in the node
+ * environment.
+ */
+env.DYNAMIC_IMPORTS_DISABLED = true;
+env.FUSE_TEST_TIMEOUT = 5000;
+env.FUSEBOX_DIST_ROOT = projectRoot;
+env.FUSEBOX_VERSION = '2.4.0';
+env.LOGGING = false;
+env.PROJECT_NODE_MODULES = join(projectRoot, 'node_modules');
+env.SPARKY_LOG = false;
+
+/*
+ * A --file flag can be used to override a default test mask.
+ */
+const maybeFileFlag = argv[2];
+const fileFlag = typeof maybeFileFlag === 'string' && maybeFileFlag.match(/^--file=(.*)/);
+
+const testFileMask = fileFlag === false ? defaultTestFileMask : fileFlag[1];
 
-let file = process.argv[2];
-let special = file && file.match(/^--file=(.*)/);
-let mask = "*.test.ts";
-if (special) {
-	mask = special = special[1];
-}
 const fuse = FuseBox.init({
-	homeDir: "src",
-	target: "server@esnext",
+	homeDir: 'src',
```

Comment:
double quotes please ;-) Let's make sure it's in the .editorconfig too

Reply:
We can just merge prettier first ;)